Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/GatewayService.java

```diff
@@ -124,6 +127,8 @@ public static GatewayService getInstance() {
      */
     public boolean discoverPatient(final PatientSearchBean patientQuerySearch) {
 
+        assertion = addUserInfoAndPurposeOf(assertion, patientQuerySearch.getUser(), patientQuerySearch.getSelectedPurposeOf(),
+                patientQuerySearch.getPurposeOfDescription());
         
         // Create the patient bean that needs to be passed to the service layer
         final gov.hhs.fha.nhinc.patientdiscovery.model.Patient patientBean
```

Comment:
Optional: Why we define all package class here?

Reply:
We are using 2 different Patient classes in Gateway service. We have an import  **import gov.hhs.fha.nhinc.admingui.model.Patient;**  .. if we don't use the package name with Patient class at line 130, it will use the Patient class listed in import which will cause compile time errors.